contrib/rpm/buildah.spec

```diff
@@ -26,7 +26,7 @@
 
 Name:           buildah
 # Bump version in buildah.go too
-Version:        1.2
+Version:        1.3
 Release:        2.git%{shortcommit}%{?dist}
```

Comment:
This was also bumped to 2.git by nalind after the shadow-utils change in the Requires. I think it should be bumped back to 1.git with the new version number.